MapboxAndroidDemo/proguard-rules.pro

```diff
@@ -121,13 +121,17 @@
 -keep class android.arch.lifecycle.** { *; }
 -keep class com.mapbox.android.core.location.** { *; }
 -keep class com.mapbox.mapboxsdk.** { *; }
+-keep class com.mapbox.android.gestures.** { *; }
```

Comment:
This shouldn't be necessary, there's no reflection/native code.

Reply:
From what I understand, all of those notes state that the class names are obfuscated and code would be inaccessible from another project. This, however, shouldn't cause any issue for the demo app and since the demo app is not consumed by any other project. The `-dontnote` entry for all of those notes should be enough.